Azure/sonic-buildimage
# [202012] BRCM SAI 4.3.5.1-2 Fix BRCM SAI regression due to ACL Egress Mirroring Action capability

#### Why I did it
BRCM SAI 4.3.5.1-1 picked up fix that address configurable counter resource fix had a regression where it also include changes to TD3 ACL Egress Mirroring Action capability that it is not supporting but claimed as capable.
This caused the Everflow Egress Mirroring testing into thinking TD3 is capable of handling it and attempted to test its functionality that resulted to ACL table programming failure.
Issue: (https://github.com/Azure/sonic-buildimage/issues/8671)

#### How I did it
Examined the SAI changes and found the list of capability for TD3 was changed to include Egress Mirroring Action.
Removed it from the list of enums and built SAI 4.3.5.1-2 and verified that the everflow testcase now skips it due to the capability is back to correct setting (Egress Mirroring Action not there any more after the cahnge)

#### How to verify it
See Issue (https://github.com/Azure/sonic-buildimage/issues/8671) for details on how to verify the fix.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog
[202012] BRCM SAI 4.3.5.1-2 Fix BRCM SAI regression due to ACL Egress Mirroring Action capability

#### A picture of a cute animal (not mandatory but encouraged)
